YouTubeSearch/src/redux/reducers/searchedVideosReducer.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  SEARCHED_VIDEO_FAIL,
+  SEARCHED_VIDEO_REQUEST,
+  SEARCHED_VIDEO_SUCCESS,
+} from '../actionType';
+
+const searchedVideosReducer = (
+  state = {
+    loading: true,
+    videos: [],
+  },
+  action: any,
+) => {
+  const { payload, type } = action;
+
+  switch (type) {
+    case SEARCHED_VIDEO_REQUEST:
+      return {
+        ...state,
+        loading: true,
+      };
+    case SEARCHED_VIDEO_SUCCESS:
+      return {
+        ...state,

```

Comment:
```suggestion
          videos: payload,
          loading: false,
```
Если ты переопределяешь весь `state`, то не нужно делать его деструктуризацию